content/test/integration/service/deployments/deployment-sorting.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { loadTestEnvironment } from "../../E2ETestEnvironment";
+import { buildDeployData } from "../../E2ETestUtils";
+import { buildDeployment, buildEvent } from "../../E2EAssertions";
+import assert from "assert";
+import { TestServer } from "../../TestServer";
+import ms from "ms";
+import { SortingField, SortingOrder, Timestamp } from "dcl-catalyst-commons";
+
+/**
+ * This test verifies that all deployment sorting params are working correctly
+ */
+describe("Integration - Deployment Filters", () => {
+  const SYNC_INTERVAL: number = ms("1s");
+  const testEnv = loadTestEnvironment();
+  let server: TestServer;
+
+  beforeEach(async () => {
+    [server] = await testEnv.configServer(SYNC_INTERVAL).andBuildMany(3);
+  });
+
+  it(`When getting all deployments without sortby then the order is by local and desc`, async () => {
+    // Start server
+    await Promise.all([server.start()]);
+
+    // Prepare data to be deployed
+    await deployToServer(server);
+    await deployToServer(server);
+    await deployToServer(server);
+
+    const deployments = await server.getDeployments();
+    assert.equal(3, deployments.length, `Expected to find 3 deployments on server ${server.getAddress()}. Instead, found ${deployments.length}.`);
+
+    // Make sure that deployments are sorted per descending local timestamp
+    for (let i = 1; i < deployments.length; i++) {
+      assert.ok(deployments[i - 1].auditInfo.localTimestamp > deployments[i].auditInfo.localTimestamp);
+    }
```

Comment:
It could be nice to have a function that looks something like this:
```
function assertSortedBy(field: SortingField, order: SortingOrder) 
```

That way, the test is super short and clear. The complexity of how to verify the sorting can be hidden